src/gan/gen_pyt/asdl/__init__.py

```diff
@@ -0,0 +1,10 @@
+# import six
```

Comment:
cool to have all this commented out? is the `__init__.py` even needed?

Reply:
We need the file, but I don't believe it needs to have anything in it. I haven't confirmed that yet, though, so I commented it for now before fully deleting.